snapcraft/plugins/roscore.py

```diff
@@ -67,6 +68,8 @@ def build(self):
 
         os.chmod(service_wrapper, 0o755)
 
+        self._finish_build()
```

Comment:
This feels like a bad name. It doesn't say what it does, it has many statements and no docstring.
Maybe something like _patch_snap_python? I'm not sure if that's better.